### PR TITLE
PPR with relevance feedback added

### PR DESCRIPTION
**Pull request by vibnimit**
Please review it :)

### CODE REVIEW

**Review by Kunal30 — DISMISSED** [review]
Looks good to me! Approved

File: src/tasks/task6.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def execute_task6(request):
         print('PPR')
         init_ranking,Vt=rf.get_init_ranking(obj_feature_matrix=obj_feature_matrix,q=q)
         return render(request, 'visualize_images.html', {'images': init_ranking, "from_task": "task6", "rel_type": rel_feedback_type,"q":q_name, "t":m})
-        return        
+        return
```
Comment (skuma149) [inline src/tasks/task6.py]: two return statements?

Comment (vibnimit) [reply src/tasks/task6.py]: Good point removed

**Review by AdityaGovardhan — APPROVED** [review]
Looks good to merge

**Review by Thek10patil — APPROVED** [review]

**Comment by vibnimit** [comment]
Approved so rebase and merging 👍

**Pull request merged**